plugins/woocommerce/src/Internal/Orders/OrderAttributionController.php

```diff
@@ -394,28 +394,31 @@ private function log( string $message, string $method, string $level = WC_Log_Le
 	 * @return void
 	 */
 	private function send_order_tracks( array $source_data, WC_Order $order ) {
-		$origin_label        = $this->get_origin_label(
+		$origin_label = $this->get_origin_label(
 			$source_data['type'] ?? '',
 			$source_data['utm_source'] ?? '',
 			false
 		);
-		$customer_identifier = $order->get_customer_id() ? $order->get_customer_id() : $order->get_billing_email();
-		$customer_info       = $this->get_customer_history( $customer_identifier );
-		$tracks_data         = array(
-			'order_id'             => $order->get_id(),
-			'type'                 => $source_data['type'] ?? '',
-			'medium'               => $source_data['utm_medium'] ?? '',
-			'source'               => $source_data['utm_source'] ?? '',
-			'device_type'          => strtolower( $source_data['device_type'] ?? '(unknown)' ),
-			'origin_label'         => strtolower( $origin_label ),
-			'session_pages'        => $source_data['session_pages'] ?? 0,
-			'session_count'        => $source_data['session_count'] ?? 0,
-			'order_total'          => $order->get_total(),
-			// Add 1 to include the current order (which is currently still Pending when the event is sent).
-			'customer_order_count' => $customer_info['order_count'] + 1,
-			'customer_registered'  => $order->get_customer_id() ? 'yes' : 'no',
+
+		$tracks_data = array(
+			'order_id'            => $order->get_id(),
+			'type'                => $source_data['type'] ?? '',
+			'medium'              => $source_data['utm_medium'] ?? '',
+			'source'              => $source_data['utm_source'] ?? '',
+			'device_type'         => strtolower( $source_data['device_type'] ?? '(unknown)' ),
+			'origin_label'        => strtolower( $origin_label ),
+			'session_pages'       => $source_data['session_pages'] ?? 0,
+			'session_count'       => $source_data['session_count'] ?? 0,
+			'order_total'         => $order->get_total(),
+			'customer_registered' => $order->get_customer_id() ? 'yes' : 'no',
+		);
```

Comment:
Can we add this code via a different PR and merge it to the trunk for fix release consideration. Seems like this will prevent checkout for sites with customers having large number of orders, see #44105

Reply:
👉🏻  https://github.com/woocommerce/woocommerce/pull/44359